plugins/sass_test.ts

```diff
@@ -2,24 +2,24 @@ import { assertEquals } from 'https://deno.land/std@0.83.0/testing/asserts.ts'
 import plugin from './sass.ts'
 
 Deno.test('project scss loader plugin', async () => {
-    const { code, loader } = await plugin.transform(
+    const { code } = await plugin.precompile!(
         (new TextEncoder).encode('$someVar: 123px; .some-selector { width: $someVar; }'),
         'test.scss'
     )
     assertEquals(plugin.test.test('test.sass'), true)
     assertEquals(plugin.test.test('test.scss'), true)
+    assertEquals(plugin.loader, 'css')
     assertEquals(plugin.acceptHMR, true)
     assertEquals(code, '.some-selector {\n  width: 123px;\n}')
-    assertEquals(loader, 'css')
 })
 
 Deno.test('project sass loader plugin', async () => {
-    let ret = await plugin.transform(
+    let ret = await plugin.precompile!(
         (new TextEncoder).encode('$someVar: 123px\n.some-selector\n  width: 123px'),
         'test.sass'
     )
     assertEquals(ret.code, '.some-selector {\n  width: 123px;\n}')
-    ret = await plugin({ indentType: 'tab', indentWidth: 2 }).transform(
+    ret = await plugin({ indentType: 'tab', indentWidth: 2 }).precompile!(
```

Comment:
Why the `!` after `precompile`? is it a typo? I don't think that is valid syntax

Reply:
now loader plugin's precompile method is optional, but sass one is not